src/test/components/TrackNetwork.test.js

```diff
@@ -64,12 +69,31 @@ describe('timeline/TrackNetwork', function() {
     window.dispatchEvent(new Event('resize'));
     expect(getContextDrawCalls().length > 0).toBe(true);
   });
+
+  it('draws differently a request and displays a tooltip when hovered', () => {
+    const { getContextDrawCalls } = setup();
+    // Flush out any existing draw calls.
+    getContextDrawCalls();
+
+    const canvas = ensureExists(document.querySelector('canvas'));
+    fireEvent(
+      canvas,
+      getMouseEvent('mousemove', {
+        pageX: 12,
+        pageY: 2,
+      })
+    );
+
+    const tooltip = screen.getByTestId('tooltip');
+    expect(tooltip).toMatchSnapshot();
+
+    const drawCalls = getContextDrawCalls();
+    // We draw at least one hovered request with a different stroke style.
+    expect(drawCalls).toContainEqual(['set strokeStyle', '#0069aa']);
```

Comment:
I wonder if we should  also have a snapshot test for the drawcalls. But not sure if they are worth it. What do you think?

Reply:
yeah I can add one, to make sure that the different stroke style comes last.